Mathlib/Data/Finmap.lean

```diff
@@ -417,11 +417,13 @@ theorem insert_toFinmap (a : α) (b : β a) (s : AList β) :
     insert a b (AList.toFinmap s) = AList.toFinmap (s.insert a b) := by
   simp [insert]
 
-theorem insert_entries_of_neg {a : α} {b : β a} {s : Finmap β} :
+theorem insert_entries_of_not_mem {a : α} {b : β a} {s : Finmap β} :
```

Comment:
Arguably this should be called `entries_insert_...` not `insert_entries_...`, and if we're going to rename and deprecate we may as well go all the way.

@digama0, can I check you agree with my suggestion here, since you're an author of the file?

Reply:
Agreed. We used to not be as clear about the prefix convention, but I think we should enforce it more consistently.